package-lock.json

```diff
@@ -11125,7 +11125,7 @@
 		"isarray": {
 			"version": "0.0.1",
 			"resolved": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz",
-			"integrity": "sha512-D2S+3GLxWH+uhrNEcoh/fnmYeP8E8/zHl644d/jdA0g2uyXvy3sb0qxotE+ne0LtccHknQzWwZEzhak7oJ0COQ==",
+			"integrity": "sha1-ihis/Kmo9Bd+Cav8YDiTmwXR7t8=",
```

Comment:
Got it, I appreciate your feedback. It seems that I made a mistake with my merge which impacted the configurations on `lerna-package-lock.json` from `server/routerlicious` as it fails build on the CI. 

I'm still trying to figure out what the problem is, and one of the thoughts that I had was to install `eslint` (which I later removed as it changed 72 files) since I had the following error 

```
lerna ERR! npm run lint exited 2 in '@fluidframework/server-services-core'

lerna WARN complete Waiting for 18 child processes to exit. CTRL-C to exit immediately.

npm ERR! code ELIFECYCLE

npm ERR! errno 2

npm ERR! root@0.0.1 lint: `lerna run lint --no-sort --stream`

```

Reply:
The state at d70840fc034835ad08fdd1b3836828fa24b20093 looks like what I would have expected, and it appears to have passed CI?

Reply:
Yes, I initially tried to merge the state at `d70840f` after resolving a merge conflict.